zetaclient/zetacore/constant.go

```diff
@@ -24,6 +24,9 @@ const (
 	// PostVoteInboundMessagePassingExecutionGasLimit is the gas limit for voting on, and executing ,observed inbound tx related to message passing (coin_type == zeta)
 	PostVoteInboundMessagePassingExecutionGasLimit = 4_000_000
 
+	// PostVoteInboundCallOptionsGasLimit is the gas limit for inbound call options
+	PostVoteInboundCallOptionsGasLimit uint64 = 1_500_000
```

Comment:
What is this, why is it needed?

Reply:
I moved the magic number from `evm/observer/v2_inbound.go` to here as a constant, which can then be reused by Bitcoin as well (in V2). wdyt